src/loaders/express.js

```diff
@@ -55,6 +56,34 @@ module.exports = async (app) => {
     AWSXRay.getSegment().addMetadata('podName', config.podName);
     next();
   });
+  const authenticationEnabled = false;
+  if (authenticationEnabled) {
+    app.use(async (req, res, next) => {
+      let workRequest = [];
+      try {
+        // this try is so that the api doesn't crash if there's nothing in the body
+        workRequest = JSON.parse(JSON.parse(req.body).Message).request || false;
+      // eslint-disable-next-line no-empty
+      } catch (err) {}
```

Comment:
If there are errors, don't you want to raise it?

Reply:
Here this would error if its not a socket io request. In that case the authorization header is in the Message.request.extraHeaders